src/common/utils/validator.js

```diff
@@ -217,18 +217,31 @@ function currentOrFutureDate(dObj) {
   return false;
 }
 
-// function dateTooFarInFuture(dObj){
-//   if (dObj === null || dObj === undefined) return false;
+function dateNotTooFarInFuture(dObj) {
```

Comment:
Please can we add a docstring for what this function does

Reply:
will do